client/components/phone-input/index.jsx

```diff
@@ -109,13 +111,39 @@ const PhoneInput = React.createClass( {
 		event.preventDefault();
```

Comment:
On line 106 (sorry, GitHub won't let me put a comment there >_>), I think it should be OK to compare `this.props.value` instead (`value === this.props.value`). It should always (might need to implement `componentWillReceiveProps` or something to achieve that) have the current value (before this event, of course), so no need for an additional expensive `format` call.